deploy/cr.yaml

```diff
@@ -503,3 +503,11 @@ spec:
 #        storageName: s3-us-west
 #        compressionType: gzip
 #        compressionLevel: 6
+#      - name: weekly-s3-us-west-physical
+#        enabled: false
+#        schedule: "0 5 * * 0"
+#        keep: 5
+#        type: physical
```

Comment:
If I understand correctly here it should still be `backupType`?

Reply:
no, we converted to `type` everywhere: https://github.com/percona/percona-server-mongodb-operator/pull/1112#discussion_r1101210827